sql/src/main/java/io/crate/analyze/KillAnalyzedStatement.java

```diff
@@ -21,11 +21,18 @@
 
 package io.crate.analyze;
 
+import io.crate.sql.tree.KillStatement;
+
 public class KillAnalyzedStatement implements AnalyzedStatement {
 
-    public static final KillAnalyzedStatement INSTANCE = new KillAnalyzedStatement();
+    private final KillStatement node;
+
+    public KillAnalyzedStatement(KillStatement node) {
+        this.node = node;
```

Comment:
store the UUID here. the analyzer should do this, and trow an exception if it is not a valid uuid.
this will also make the code in the planner shorter